.eslintrc

```diff
@@ -279,6 +279,12 @@
     "@typescript-eslint/adjacent-overload-signatures": ["error"],
     "@typescript-eslint/no-extra-non-null-assertion": ["error"],
     "@typescript-eslint/no-misused-new": ["error"],
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "args": "none"
```

Comment:
Why we don't check args?

Reply:
I will enable this check in a separate PR. Thanks!

Reply:
Make sure that we have something to track this task! (Maybe any existing github issue or a new one!)

Reply:
Thanks for the info. Can you please share the issue link here so that we can mark this thread resolved! 

[For this kinda comments always reply with the issue link else the review will ask for the link and it will take time to resolve a thread!]

Reply:
This is the https://github.com/oppia/oppia/issues/8423. It is the third check under JS/TS section.